src/allmydata/web/directory.py

```diff
@@ -1290,7 +1298,7 @@ def render_HTML(self, req):
                      + stats.get("size-mutable-files", 0)
                      + stats.get("size-directories", 0))
             output += "size: %d\n" % total
-        return output
+        return output.encode("utf-8")
```

Comment:
Maybe this isn't necessary given the change to `MultiFormatResource` (not that it really seems to hurt, either).

Reply:
Probably did this before the other, yeah.